frontend/src/App.js

```diff
@@ -1,9 +1,9 @@
-import React, { useEffect } from "react";
+import React from "react";
 import HomePage from "./HomePage";
 import SignUpPage from "./SignUpPage";
-import axios from "axios";
 import { BrowserRouter, Switch, Route } from "react-router-dom";
 import FarmSearchLocations from "./FarmSearchLocations";
+import { createTheme, ThemeProvider } from "@mui/material";
 
 export default function App() {
     const listOfFarms = [
```

Comment:
As soon as a merge my change, we can start making requests from the DB to get this info.